cityofaustin/atd-vz-data
# Data model: locations list and details pages

## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/17443
- https://github.com/cityofaustin/atd-data-tech/issues/17444

## Testing

1. Follow the steps in https://github.com/cityofaustin/atd-vz-data/pull/1440 to start your local stack, including running the CRIS import. 

2. After you run the CRIS import, use your SQL client to refresh the location list view:

```sql
refresh materialized view locations_list_view;
```

3. In the VZE, test out the locations list by sorting all columns, searching by location ID, and searching by street name

4. Inspect the location details page, ideally using a location that has a high number of fatalities and crashes. Make sure each of the cards are rendering correctly. Do some rough cross checking that the various metric cards are lining up with the crashes associated with the location. Test the search/filtering functions on the CR3 and Non-CR3 crashes tables. Add a location note.

5. Test the "Download Global Data" button. It should work, but it is still pointed at the old `atd_txdot_crashes` table. We will fix that here: https://github.com/cityofaustin/atd-data-tech/issues/17944.

6. Head over to the crashes list and crash details pages and make sure those are loading normally 👍 

---
#### Ship list
